spec/spec_helper.rb

```diff
@@ -20,7 +20,7 @@
   config.mock_with :rspec do |mocks|
     mocks.verify_partial_doubles = true
   end
-
+  config.include UserHeaderSpecHelper
```

Comment:
can you move this to the top of the configure block so that it stands out a bit?  Here in the middle it's sort of buried amongst the boilerplate configuration and you can't see it.